### PR TITLE
Add UX WG to WORKING-GROUPS.md

### PR DESCRIPTION
**Pull request by csantanapr**
# Changes
Add UX WG to WORKING-GROUPS.md
Related to #466

### CODE REVIEW

**Comment by csantanapr** [comment]
cc @knative/technical-oversight-committee 

/assign @evankanderson @mattmoor

File: working-groups/WORKING-GROUPS.md [hunk 1]
```diff
@@ -297,7 +297,25 @@ Security concerns across Knative components
...
+| Meeting Notes              | [Notes](https://docs.google.com/document/d/1NSlGHen5Dh6c2A0LGavGWibddWrlSLV_PbEbMpNjSTU/edit?usp=sharing)                                               |
+| Document Folder            | [Folder](https://drive.google.com/drive/folders/1XzZGqV7yHo38d_l7rH1uSIrbQp3JlbBP?usp=sharing)                                                          |
+| Slack Channel              | [#user-experience](https://slack.knative.dev/messages/user-experience)                                                                                  |
+| Github Team WG leads       | [@knative/ux-wg-leads](https://github.com/orgs/knative/teams/ux-wg-leads/members)                                                                       |
+
+| &nbsp;                                                        | Leads           | Company | Profile                                             |
+| ------------------------------------------------------------- | --------------- | ------- | --------------------------------------------------- |
+| <img width="30px" src="https://github.com/omerbensaadon.png"> | Omer Bensaadon  | VMware  | [evankanderson](https://github.com/omerbensaadon)   |
```
Comment (mattmoor) [inline working-groups/WORKING-GROUPS.md:317]: Wrong name on profile

Comment (csantanapr) [reply working-groups/WORKING-GROUPS.md:317]: duh I will fix

**Comment by csantanapr** [comment]
@mattmoor I fixed the profile name for Omer, take a look

**Review by rhuss — COMMENTED** [review]
/lgtm
/approve

**Comment by csantanapr** [comment]
Thank you @rhuss for the review

**Comment by csantanapr** [comment]
Oh weird that @rhuss doesn't have karma to approve now that he is TOC member

**Comment by csantanapr** [comment]
Woot ! 🚀

**Pull request merged**